orchagent/aclorch.cpp

```diff
@@ -807,6 +808,13 @@ bool AclRule::validateAddMatch(string attr_name, string attr_value)
         {
             return false;
         }
+        else if (attr_name == MATCH_TUNNEL_TERM)
+        {
+            if (attr_value != "true")
+            {
+                matchData.enable = false;
```

Comment:
matchData.enable = (attr_value == "true")?

Reply:
This is already set here. https://github.com/vivekrnv/sonic-swss/pull/32/files#diff-d97175d30783db8709cc74eb41e215bf958bcf304be11d836830b03cd29e917cR803

Reply:
I see that. But still, I suggest making the code compact and removing unneeded lines.

Reply:
Hmm, will update

Reply:
updated